ovos_utils/file_utils.py

```diff
@@ -18,6 +18,24 @@
 from ovos_utils.system import search_mycroft_core_location
 
 
+def __HACK_preload():  # TODO - remove me soon, make resolve_ovos_resource_file FAST
+    """ there is some latency on initial wake word detection without this,
+     checks several ovos packages and needs to import them,
+     this method is speeding things up as a workaround,
+    main culprit is ovos_workshop which causes a scan of all OCP plugins (bug ?)"""
```

Comment:
This is related to top-level `init` imports that import an OCP submodule that includes a load of all plugin on import. Could this be moved inside the listener to avoid extra latency for any file_utils imports?

Reply:
ovos_workshop itself should be fixed, this impacts anything needing the resolve_resource_file methods thats why i put it here instead of listener (i had it there originally)